_topic_map.yml

```diff
@@ -2981,6 +2981,24 @@ Topics:
   # Tracing
   - Name: Tracing requests using Jaeger
     File: serverless-tracing
+    # Custom domains
+  - Name: Mapping a custom domain name to a Knative service
+    File: serverless-domain-mapping
+    # Routes
+  - Name: Configuring routes for Knative services
+    File: serverless-configuring-routes
+# Service mesh integration
+- Name: Service Mesh integration
+  Dir: service_mesh
+  Topics:
+  - Name: Integrating Service Mesh and OpenShift Serverless with Kourier enabled
+    File: serverless-ossm-with-kourier
+  - Name: Configuring TLS for a custom domain using Service Mesh and Kourier
+    File: serverless-ossm-tls-with-kourier
+  - Name: Configuring custom domains for Knative services with Service Mesh and Kourier
+    File: serverless-ossm-custom-domains
+  - Name: Integrating Service Mesh and OpenShift Serverless without Kourier
+    File: serverless-ossm-without-kourier
```

Comment:
From the docs drafts I got, mTLS is _part of _ integrating service mesh and serverless without kourier, is this not correct @markusthoemmes ?
i.e. you cannot use mTLS with kourier, but it's something you have to do to integrate SM+Serverless without using Kourier?

Reply:
Yes you cannot use mTLS with Kourier
No it's not something you **have** to do when integrating SM and Serverless "natively" (without Kourier). The native integration only **enables** you to do it, you don't have to.

Reply:
I'm reshuffling this a bit to see if I can improve it.
@markusthoemmes something I'm not clear on is where in the "native" / new integration docs the basic setup ends and the mTLS stuff begins...

Reply:
I guess there is no basic stuff right now. The draft was structured around including mTLS.

The "basic" stuff is essentially enabling `istio` in the KnativeServing resource (without even specifying the deployment section and the additional annotations and stuff). That'd be "the most basic way of using Knative with ServiceMesh".

We're assuming here, that nobody would do that if they don't also want to use mTLS, hence we're pushing in that direction right away. Otherwise the customer might ask: Why would I even not just use the defaults (i.e. Kourier).